swaps/swaps_service.go

```diff
@@ -1115,6 +763,448 @@ func (svc *swapsService) ListSwaps() ([]Swap, error) {
 	return swaps, nil
 }
 
+func (svc *swapsService) subscribePendingSwaps() {
+	var swaps []db.Swap
+	if err := svc.db.Where("state = ?", constants.SWAP_STATE_PENDING).Find(&swaps).Error; err != nil {
+		logger.Logger.WithError(err).Error("failed to load pending swaps")
+		return
+	}
+	if len(swaps) == 0 {
+		return
+	}
+
+	logger.Logger.WithField("count", len(swaps)).Info("Resuming pending swaps...")
+
+	boltzWs := svc.boltzApi.NewWebsocket()
+	if err := boltzWs.Connect(); err != nil {
+		logger.Logger.WithError(err).Error("Could not connect to Boltz websocket")
+		return
```

Comment:
do we really want a return here? then the user is stuck if they have pending swaps